### PR TITLE
vim-patch:partial:8a3b805c6c9c

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/8a3b805c6c9cae341d560df9c3567ebbe42a7404

skip builtin.txt (needs 8.2.4838)
skip message.txt (whitespace changes)

### CODE REVIEW

**Review by zeertzjq — COMMENTED** [review]
`expr-[:]` change is applicable

**Comment by clason** [comment]
I couldn't find `expr10` anywhere in our repo?

**Comment by zeertzjq** [comment]
But you can find `expr-[:]`

File: runtime/doc/options.txt [hunk 1]
```diff
@@ -3799,9 +3799,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 <							*lcs-leadmultispace*
 	  leadmultispace:c...
 			Like multispace value, but only for leading whitespace
-			Overrides |lcs-lead| for leading multiple spaces.
-			`:set listchars=leadmultispace:---+` shows ten consecutive
-			leading spaces as:
+			overrides |lcs-lead| for leading multiple spaces.
```
Comment (seandewar) [inline runtime/doc/options.txt:3802]: I think what is actually needed is a  `.` at the end of the first line, like:
```vimhelp
			Like multispace value, but only for leading
			whitespace. Overrides |lcs-lead| for leading multiple
			spaces. `:set listchars=leadmultispace:---+` shows ten
			consecutive leading spaces as:
```
Though it does cause a bit of wrapping.

File: runtime/doc/editing.txt [hunk 2]
```diff
@@ -1173,7 +1173,7 @@ Examples: >
 
 If you want to always use ":confirm", set the 'confirm' option.
 
-			*:browse* *:bro* *E338* *E614* *E615* *E616*
+					*:browse* *:bro* *E338* *E614* *E615* *E616*
```
[the remark this answers is not in the document]
Comment (clason) [reply runtime/doc/editing.txt:1176]: I have to rely on you to tell me exactly what to change here; I can just port the changes to the best of my ability (which is limited enough...)

[the remark this answers is not in the document]
Comment (seandewar) [reply runtime/doc/editing.txt:1176]: For now yeet the E6XX errors (they're definitely gone, and in Vim too)

**Pull request merged**